Setup32/Setup32.vdproj

```diff
@@ -535,7 +535,7 @@
             {
             "AssemblyRegister" = "3:1"
             "AssemblyIsInGAC" = "11:FALSE"
-            "AssemblyAsmDisplayName" = "8:WasatchNET, Version=2.2.8.0, Culture=neutral, processorArchitecture=x86"
+            "AssemblyAsmDisplayName" = "8:WasatchNET, Version=2.2.9.0, Culture=neutral, processorArchitecture=x86"

```

Comment:
Are we on 9 or 10?

Reply:
I saw that too...I'm honestly not sure how that happened.  The "AssemblyInfo.cs" file had the correct value.  I'll check in the merge.

Reply:
Gah.  That 2.2.8 -> 2.2.9 referred to the version of LibUsbDotNet.dll, NOT Wasatch.NET, even though we were changing between almost the exact same version numbers :-(